### ISSUE TITLE
Possibility to use Adwaita

### ISSUE THREAD

**Issue by TacHawkes** [JuliaGtk/Gtk4.jl#10]
Hi,

I have come across the Adwaita/libadwaita (https://gnome.pages.gitlab.gnome.org/libadwaita/doc/1-latest/) design theme for Gtk from the GNOME project. Is it possible to use the libadwaita library for Gtk with the Julia package in some way?

**Comment by jwahlstrand** [comment]
It should be with a bit of work. The first step is to add libadwaita to Yggdrasil so we have a JLL. I started this a few months ago but couldn't get it to compile for MacOS. I'll take a look at it again.

In principle once that is done, creating bindings should be pretty straightforward. I'd be inclined to include them in this repo.

**Comment by jwahlstrand** [comment]
The basics are now in the directory "Adwaita", including an example. I have not used it a whole lot myself, but it seems to work. I could register it but I think it should have some documentation first.

**Comment by TacHawkes** [comment]
Thanks for tackling this, on MacOS I get this error: ERROR: could not load library "libadwaita-1.so.0"

Should I open a new issue?

**Comment by jwahlstrand** [comment]
Oops, I see the problem but I won't be able to fix it until later this week.

**Comment by jwahlstrand** [comment]
OK, see if it works now.

**Comment by TacHawkes** [comment]
Yes, it work's now! I've tested it with your example.

Thank you very much :)
